source/AI.cpp

```diff
@@ -2429,7 +2429,7 @@ void AI::Attack(Ship &ship, Command &command, const Ship &target)
 	// Have a 10% minimum to avoid ships getting in a chase loop.
 	const bool isAbleToRun = target.MaxVelocity() * SAFETY_MULTIPLIER < ship.MaxVelocity();
 
-	ShipAICache &shipAICache = ship.GetAICache();
+	const ShipAICache &shipAICache = ship.GetAICache();
```

Comment:
I'm not sure this change is necessary, but I'm fine with leaving it here if it compiles with the latest master.

Reply:
Cool. Well, it compiles.